csi: make storage class updated with csi-users in external cluster (#10278)
rook/rook: Documentation/CRDs/ceph-cluster-crd.md

    rook-ceph-external   /var/lib/rook                162m   Connected   HEALTH_OK
    ```

> 6. StorageClass will also be created, verify it's creation.

Reviewer: ```suggestion
6. StorageClass will also be created, verify its creation.
```

```console

> kubectl -n rook-ceph-external  get sc

Reviewer: ```suggestion
kubectl -n rook-ceph-external get sc
```

```

> For cephFS storageclass you also need to export CEPHFS_FS_NAME,CEPHFS_POOL_NAME or you can pass this parameters with a CLI flag(--cephfs-data-pool-name,--cephfs-filesystem-name) while running the python script.

Reviewer: ```suggestion
For CephFS StorageClass you also need to export `CEPHFS_FS_NAME`, `CEPHFS_POOL_NAME` or you can pass these parameters with a CLI flag (--cephfs-data-pool-name,--cephfs-filesystem-name) while running the script.
```
